packages/mdc-textfield/package.json

```diff
@@ -6,7 +6,7 @@
   "keywords": [
     "material components",
     "material design",
-    "textfield"
+    "text field"
```

Comment:
Nice catch 👍

Reply:
We probably want both "textfield" and "text field"